pkgs/by-name/tt/tts/package.nix

```diff
@@ -17,15 +17,15 @@ let
   };
 in
 python.pkgs.buildPythonApplication rec {
-  pname = "tts";
-  version = "0.20.2";
+  pname = "coqui-tts";
+  version = "0.25.1";
   pyproject = true;
 
   src = fetchFromGitHub {
-    owner = "coqui-ai";
-    repo = "TTS";
+    owner = "idiap";
```

Comment:
@eginhard do you consider this a maintained fork of TTS?

Reply:
Yes, we will continue to maintain it.